### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/owncast/commit/c576bb47c6f046b905c8b9d5c4f7f73911ad0378#commitcomment-89084355

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/owncast/commit/5120615b09bf43577fab8972adf301f8ef7233cc

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: .github/workflows/javascript-formatting.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Format Javascript
+name: Format JavaScript
```
Comment (jsoref) [inline .github/workflows/javascript-formatting.yml:1]: Brand

File: activitypub/apmodels/actor.go [hunk 2]
```diff
@@ -53,9 +53,9 @@ type ExternalEntity interface {
 	GetW3IDSecurityV1PublicKey() vocab.W3IDSecurityV1PublicKeyProperty
 }
 
-// MakeActorFromExernalAPEntity takes a full ActivityPub entity and returns our
+// MakeActorFromExternalAPEntity takes a full ActivityPub entity and returns our
 // internal representation of an actor.
-func MakeActorFromExernalAPEntity(entity ExternalEntity) (*ActivityPubActor, error) {
+func MakeActorFromExternalAPEntity(entity ExternalEntity) (*ActivityPubActor, error) {
```
Comment (jsoref) [inline activitypub/apmodels/actor.go:58]: It's possible this is a public API...

File: controllers/admin/config.go [hunk 3]
```diff
@@ -285,7 +285,7 @@ func SetNSFW(w http.ResponseWriter, r *http.Request) {
 	controllers.WriteSimpleResponse(w, true, "changed")
 }
 
-// SetFfmpegPath will handle the web config request to validate and set an updated copy of ffmpg.
+// SetFfmpegPath will handle the web config request to validate and set an updated copy of ffmpeg.
```
Comment (jsoref) [inline controllers/admin/config.go:288]: brand

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine

**Comment by jsoref** [comment]
@gabek anything I can do to push this along?

**Comment by gabek** [comment]
I was under the impression this was just a bot spamming repos trying to get people to use a Github action and when I saw it renaming function names I figured i'd ignore it. But it seems like maybe you're a real person. I'll give it a look.

**Comment by jsoref** [comment]
Sorry. I'm very much a real person.

**Comment by jsoref** [comment]
@gabek if I rebase, will you look?

**Comment by rmens** [comment]
Except for some function names that might cause breaking things, this PR seems to point out valid mistakes and typos. Not the most critical stuff, but it would be a shame if this got closed unmerged.

**Comment by gabek** [comment]
It's the mass automated function name changing that I'm very uncomfortable with. I'm fine with fixing spelling, but a bulk find and replace of strings by an automated tool that is changing functional code isn't the approach I feel ok with. If somebody wanted to put in a PR saying they fixed the spelling of a function name that's great, but this isn't that.

**Comment by jsoref** [comment]
I'm happy to split this however you like, but I wouldn't want to be misinterpreted. I review all the changes I propose, whether it's a single commit w/ a single character change or a thousand commit change. It's more common for me to split PRs that are much larger, but I'm flexible.

Would you be happier if the PR excluded these commits:
https://github.com/owncast/owncast/pull/2314/commits/f5c41243129459428f80a8c69a78f31a4d05e426
https://github.com/owncast/owncast/pull/2314/commits/2f00042167a92e8fb07879db71c00156c69b1834
https://github.com/owncast/owncast/pull/2314/commits/56b6849ab09d36e99af5c70b67247feebc81f7c2
https://github.com/owncast/owncast/pull/2314/commits/b8feafc33e89c6c45b14f42b496a32e312d8f64b
(Or conversely, a PR that only included them?)

**Pull request closed without merging**